src/governance/ProfitManager.sol

```diff
@@ -60,6 +60,13 @@ contract ProfitManager is CoreRef {
     /// if the loss is greater than the surplus buffer, the `creditMultiplier` is updated down.
     uint256 public surplusBuffer;
 
+    /// @notice amount of first-loss capital for a given term.
+    /// This is a number of CREDIT token held on this contract that can be used to absorb losses in
+    /// cases where a loss is reported through `notifyPnL` in a given term.
+    /// When a loss is reported in a given term, its termSuplusBuffer is donated to the general
+    /// surplusBuffer before calculating the loss.
+    mapping(address => uint256) public termSurplusBuffer;
```

Comment:
it would be good to add a comment at the top of this file explaining the seniority stack of the debt.

1. per term surplus buffer
2. global surplus buffer
3. credit holders

Reply:
Done in [04b6607](https://github.com/volt-protocol/ethereum-credit-guild/pull/31/commits/04b6607e182b45de6fc8eb1b999218f6794d4a9d)